### ISSUE TITLE
No resize event?

### ISSUE THREAD

**Issue by tester12345678910111213** [cordova-plugin-camera-preview/cordova-plugin-camera-preview#436]
One of the biggest issues with this plugin is there is no resize vent to change it's position / size on the screen. Somebody may be holding an ipad, and rotate it's orientation. It's a huge must to be able to change it's position on the screen.

**Comment by Choppel** [comment]
Hi @tester12345678910111213 
do you mean the picture that was taken, that under some circumstances has the wrong orientation?

**Comment by tester12345678910111213** [comment]
No, i mean when i rotate my device, i should be able to do these things:

- resize the camera display area on screen size change / orientation change
- change the x and y coordinates on the size / orientation change.

**Comment by Choppel** [comment]
Orientation detection is understandably not part of this plugin.
But you could use cordova-plugin-orientation for that. When this plugin detects a change in the orientation you can resize the camera preview.

I hope I could help you. If not, please specify the exact problem you have.

**Comment by Choppel** [comment]
Sorry, I thought there was a way of changing the coordinates. I am afraid there isn't. It's just the dimensions that you can change.

Why do you need to change this in the first place? Just asking because I want to understand the whole problem.
Before starting the preview I lock the screen in the orientation that it currently has and show the preview in full screen with html buttons on top. Works like a charm and feels very similar to the native camera app. So there's no need to resize or reposition the preview screen.

**Comment by StarStep** [comment]
It would be nice to have responsive preview tho...

**Comment by Et3rnal** [comment]
I Listed to `ordinationchange` and it I'm inside the camera I stop it and open it again, it takes less than a second. Not perfect but not bad workaround until someone fix it

```
window.addEventListener("orientationchange", function() {
    //if the camera is open , stopCamera() then startCamera();
});
```

**Comment by ClaudiaHestra** [comment]
I still have the problem of stopCamera(); too slow.... 
I have cordova-plugin-camera-preview 0.10.0 
cordova 8.0.0
ios 12.0

can you help me?

**Comment by Et3rnal** [comment]
I'm on android and stopCamera is fast!
`CameraPreview.stopCamera();`

**Comment by ClaudiaHestra** [comment]
not on ios....

**Comment by jacobscarter** [comment]
I run into the same issue, there are multiple problems when dealing with device rotation. You can listen to the `orientationchange` event but you need to wait for orientation to stop before triggering stop start or you get weird results. Found some help here but still is spotty in reliability: https://github.com/gajus/orientationchangeend

Next issue is as state stop start is too slow, even if you put the start inside the stops promise race conditions still exist that cause the start to fail every now and then.

A camera preview resize method would be great and solve these issues. Additional feature or work around would be great, ill update if I find one, would love to hear other thoughts.
